core/logger/redirectStderrLinux.go

```diff
@@ -0,0 +1,18 @@
+//+build linux darwin
+
+package logger
+
+import (
+	"os"
+	"syscall"
+)
+
+// redirectStderr to the file passed in
```

Comment:
Better description needed for this functions

Reply:
Changed.